README.md

```diff
@@ -64,6 +63,11 @@ The message is a concatenated string  generated from the following parts:
   * Each header is separated by a newline in the concatenated string
 * `Resource`: The HTTP request path + query string, e.g. `/resource?key=value`
 
+## Time validation
+
+It is recommended that the implementation does not except request older than 15 minutes and does not accept requests that were sent from "the future". Eg, requests with a timestamp more than 5 minutes out from the server's timestamp and not older than 15 minutes from the server's timestamp.
```

Comment:
* Change "except" to "accept"
* Change "request" to "requests"

Would also like to change the wording of the future message, but I like the general point.